src/app/all-recipes/all-recipes.component.scss

```diff
@@ -0,0 +1,98 @@
+@import "@taiga-ui/core/styles/taiga-ui-local.scss";
+
+app-all-recepies {
+  [tuiWrapper][data-appearance="edit"] {
+    background: #ffba51;
+    color: #fff;
+
+    @mixin wrapper-hover {
+      background: #ffbf00;
+    }
+
+    &:hover {
+      @include wrapper-hover;
+    }
+  }
+
+  .book-image {
+    position: absolute;
+    width: 100%;
+    background-image: url("../../assets/opened-book — kopia.png");
```

Comment:
Please change file name. I cannot contain "- kopia" in name. It's unprofessional :)